trade-finance-manager-api/src/v1/helpers/entra-group-to-tfm-role.js

```diff
@@ -0,0 +1,25 @@
+require('dotenv').config();
+const { TEAMS } = require('../../constants/teams')
+
+const getRolesGroupedByEntraId = () => {
+  const entraGroups = {};
+
+  Object.values(TEAMS).forEach(group => {
+    const azureId = process.env[group.azure_id_env_var];
+    entraGroups[azureId] = group.id;
+  }, {});
+
+  return entraGroups;
+}
+
+const getTfmRolesFromEntraGroups = (groupIds) => {
+  const entraIdMap = getRolesGroupedByEntraId();
+  const tfmRoles = groupIds.map(id => entraIdMap[id]);
+  return tfmRoles;
+}
+
+
```

Comment:
Documentation

Reply:
This is quite old comment, file was refactored and documentation JsDoc added.